test/cpp/microbenchmarks/bm_timer.cc

```diff
@@ -1,123 +0,0 @@
-/*
```

Comment:
Why is this benchmark being removed?  Don't we still care about timer performance?

Reply:
We don't run it (except as a test) so we don't look at the results, it's benchmarking the old API that we're trying to replace, and it was going to be a pain to update due to assumptions it was making about the representation of time.